### PR TITLE
Implement support to installing MariaDB subchart

### LINKED ISSUE #7: Implement internal Database Deployment
We currently have the config in the values.yaml but no dependency and deployment for an "intern" database deployment.

### PR DESCRIPTION
**Pull request by jacksgt**
By directly pulling in the Bitnami MariaDB Helm chart, users can
easily get started with the application without first needing to set
up an external database.

Fixes #7

https://artifacthub.io/packages/helm/bitnami/mariadb
https://github.com/bitnami/charts/tree/master/bitnami/mariadb

### CODE REVIEW

File: limesurvey/values.yaml [hunk 1]
```diff
@@ -33,10 +35,15 @@ mariadb:
     # MariaDB custom user name
     username: limesurvey
     # MariaDB custom user password
-    password: ""
-    # TODO: persistence
+    password: "your-super-secret-password"
+    # Re-use credentials from an existing secret
+    existingSecret: ""
```
Comment (jacksgt) [inline limesurvey/values.yaml:40]: This value is not implemented yet.

File: limesurvey/templates/deployment.yaml [hunk 2]
```diff
@@ -62,15 +62,52 @@ spec:
...
               valueFrom:
                 secretKeyRef:
                   name: {{ include "limesurvey.databaseSecretName" . }}
                   key: mariadb-password
+              {{- end }}
             - name: DB_USERNAME
+              {{- if eq .Values.mariadb.enabled true }}
+              value: {{ .Values.mariadb.auth.username }}
+              {{- else }}
               value: {{ .Values.externalDatabase.username }}
```
Comment (jacksgt) [inline limesurvey/templates/deployment.yaml:103]: Hmm, maybe I should wrap these values in a secret instead of directly accessing them...

**Comment by jacksgt** [comment]
@martialblog Wanna have a look again?

**Review by martialblog — APPROVED** [review]

**Pull request merged**